runbot/tests/test_utils.py

```diff
@@ -0,0 +1,55 @@
+from odoo.tests.common import TransactionCase, mute_logger
+
+from odoo.addons.runbot.common import pseudo_markdown
+
+
+class TestUtils(TransactionCase):
+
+    def test_md_formatting(self):
+        self.assertEqual(
+            pseudo_markdown(
+                "**strong** ~~delete~~ __italic__ \n"
+            ),
+            "<strong>strong</strong> "
+            "<del>delete</del> "
+            "<ins>italic</ins> "
+            "<br/>\n"
+        )
+
+    def test_md_icons(self):
+        self.assertEqual(
+            pseudo_markdown(
+                "@icon-star"
+            ),
+            '<i class="fa fa-star"></i>'
+        )
+
+    def test_md_urls(self):
+        # Basic
+        self.assertEqual(
+            pseudo_markdown(
+                "[name](https://runbot.odoo.com)"
+            ),
+            '<a href="https://runbot.odoo.com">name</a>'
+        )
+        # Test with attributes
+        self.assertEqual(
+            pseudo_markdown(
+                "[name](https://runbot.odoo.com){target _blank}"
+            ),
+            '<a href="https://runbot.odoo.com" target="_blank">name</a>'
+        )
+
+        # Invalid input
+        with mute_logger('odoo.addons.runbot.common'):
```

Comment:
```suggestion
        with self.assertLogs(level='WARNING'):
```
while at it, why not assert that the warning was triggered